src/live_restore/live_restore_state.c

```diff
@@ -10,11 +10,11 @@
 #include "live_restore_private.h"
 
 /*
- * __wti_live_restore_migration_complete --
+ * __wt_live_restore_migration_complete --
  *     Return if live restore is past the background migration stage.
  */
 bool
-__wti_live_restore_migration_complete(WT_SESSION_IMPL *session)
+__wt_live_restore_migration_complete(WT_SESSION_IMPL *session)
 {
     WTI_LIVE_RESTORE_FS *lr_fs = (WTI_LIVE_RESTORE_FS *)S2C(session)->file_system;
```

Comment:
Per the [other thread](https://github.com/wiredtiger/wiredtiger/pull/11813/files#r2025215613) now that this function is public it can be called when live restore isn't configured, so we need a safety check that `WT_CONN_LIVE_RESTORE_FS` is set before we cast the file system to a `WTI_LIVE_RESTORE_FS`.

When live restore isn't configured I don't think it makes sense to return a true/false value. I'd lean towards an assertion failure when that happens, but if we want to handle that case the function will need to return a `(COMPLETE|IN_PROGRESS|NOT_ENABLED)` enum instead of a bool